src/core/js/adapt.js

```diff
@@ -106,9 +106,11 @@ define([
         }
 
         var navigationHeight = $(".navigation").outerHeight();
+        // prevent scroll issue when component description aria-label coincident with top of component
+        var ariaLabelBuffer = $(selector.is('.component')) ? 5 : 0;
```

Comment:
If this offset is going to handled here, I'd prefer a more programmatical approach like below:
```js
var $element = $(selector);
var offsetTop = -navigationHeight;

if (Adapt.config.get('_accessibility')._isActive &&
    $element.hasClass('component')) {
    offsetTop -= $element.find('.aria-label').height() || 0;
}
```
Am I right in thinking it's only an issue in Firefox? If it's such a small use case that it's only when accessibility is active *and* it's a component *and* in one browser, then it may make more sense to fix in the theme.